src/pyscipopt/scip.pxi

```diff
@@ -4452,59 +7281,136 @@ cdef class Model:
 
     # Diving methods (Diving is LP related)
     def startDive(self):
-        """Initiates LP diving
-        It allows the user to change the LP in several ways, solve, change again, etc, without affecting the actual LP that has. When endDive() is called,
-        SCIP will undo all changes done and recover the LP it had before startDive
-        """
+        """Initiates LP diving.
+        It allows the user to change the LP in several ways, solve, change again, etc,
+        without affecting the actual LP that has. When endDive() is called,
```

Comment:
that has...?

Reply:
For pure comment docstrings I've just saved a line. Isn't a big change.

Reply:
Is this sentence grammatically correct? It seems like there is something missing after "that has". What does the actual LP have?